### DISCUSSION TITLE
How set print property to ocg

### DISCUSSION THREAD

**Discussion by Aleksandr-E** [pymupdf/PyMuPDF#3672] [Looking for help]
Hi! How set print property to ocg when ocg visibility set False?

**Comment by JorjMcKie** [comment]
Please be more verbose by all means! I simply don't understand what you mean.

**Comment by Aleksandr-E** [comment]
I'm sorry, my english is so bad. 
When I print the pdf file containing visible optional content group (create with command add_ocg('printed', on = True)), then it is visible when printing. But if a print pdf file containing unvisible ocg (create with command add_ocg('printed', on = False)), information from this ocg not printing. 
I need that when printing the file ocg always prints. that is, set the properties as in the attached picture
![layers-2-1](https://github.com/pymupdf/PyMuPDF/assets/73437757/c9f0ec0d-e04e-47b0-a37f-8258817396a7)

**Reply by JorjMcKie** [reply]
The link does not work (displays a black page).
But if I understand you correctly, you are saying that you want to show everything when printed and respect invisibility only when displayed?

**Reply by Aleksandr-E** [reply]
Yes

**Reply by JorjMcKie** [reply]
Ok, finally got you.
Sorry, but this is not currently supported by PyMuPDF.

**Reply by Aleksandr-E** [reply]
Thank you.
